jmx-metrics/src/main/groovy/io/opentelemetry/contrib/jmxmetrics/MBeanHelper.groovy

```diff
@@ -87,9 +104,10 @@ class MBeanHelper {
         }
 
         def ofInterest = isSingle ? [mbeans[0]]: mbeans
+
         return ofInterest.collect {
             try {
-                it.getProperty(attribute)
+                attributeTransformation.containsKey(attribute) ? attributeTransformation[attribute](it) : it.getProperty(attribute)
```

Comment:
Small point -- I'm not sure about groovy, but in java calling `containsKey()` immediately followed by `get()` could (in early versions at least) incur two lookups. It's probably fine, and the size of the number of beans in our maps here is so small to render this moot....however, if you did factor out a method at least you could avoid the code duplication below. Something like `getPropertyValue(bean, attribute)`

Reply:
Hey @breedx-splk, yep I refactored this when resolving the conflict with the callback change